src/main/java/com/onlinebookstore/store/model/Book.java

```diff
@@ -0,0 +1,27 @@
+package com.onlinebookstore.store.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
DOn't use Data from Lombok on JPA entities, it can cause problems, you can read more about it. use simple Getter and Setter for now